pkgs/by-name/ra/redbean/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  arch =
```

Comment:
unused binding

Reply:
removed (the arch part, because lib+stdenv+fetchurl are used)